fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -230,33 +230,48 @@ void FixedLagSmoother::processQueue(fuse_core::Transaction& transaction)
   auto current_time = ros::Time(0, 0);
   if (!pending_transactions_.empty())
   {
-    current_time = pending_transactions_.front_key();
+    current_time = pending_transactions_.front().stamp();
   }
   // Attempt to process each pending transaction
-  while (!pending_transactions_.empty())
+  auto sensor_blacklist = std::vector<std::string>();
+  auto transaction_iter = pending_transactions_.rbegin();
```

Comment:
Maybe rename to `transaction_riter` to signal it's a reverse iterator :thinking: